### PR TITLE
change two image heights to fit the screen, add a missing closing parenthesis 🎨

### PR DESCRIPTION
**Pull request by alirezaAsadi2018**
changed two image heights to fit the screen, added a missing closing parenthesis

### CODE REVIEW

File: slides/lecture-2/index.html [hunk 1]
```diff
@@ -1414,7 +1414,7 @@ <h2>Base64</h2>
       representation.
     </li>
   </ul>
-  <img src="./img/base64.png" />
+  <img src="./img/base64.png" height="400vh"/>
```
Comment (1995parham) [inline slides/lecture-2/index.html:1417]: I think this should go into style attribute just like the above case.

**Review by 1995parham — CHANGES_REQUESTED** [review]

File: slides/lecture-2/index.html (hunk 1 above)
Comment (alirezaAsadi2018) [reply slides/lecture-2/index.html:1417]: Sorry, I will fix that!!

**Comment by alirezaAsadi2018** [comment]
I think this line is a bit confusing and have no idea how to rephrase it🤔:
```
<li>
Client requests resource, Server challenges, Client asks User/Pass,
Client responses, Server authenticates and allows
</li>
```
**reference**: HTTP slide 61, 3rd bullet

**Comment by 1995parham** [comment]
I agree with the confusion. I am going to merge this PR, can you leave your last comment as an issue to resolve it later? You can use the reference in the new issue button from the menu.

**Review by 1995parham — APPROVED** [review]

**Pull request merged**